packages/@uppy/image-editor/src/ImageEditor.tsx

```diff
@@ -77,14 +141,18 @@ export default class ImageEditor extends UIPlugin {
     return false
   }
 
-  save = () => {
-    const saveBlobCallback = (blob) => {
+  save = (): void => {
+    const saveBlobCallback = (blob: Blob | null) => {
```

Comment:
Why type `blob` as `Blob | null` to then throw a runtime error when it's null?
```suggestion
    const saveBlobCallback = (blob: Blob) => {
```

Reply:
Otherwise it's a type mismatch with where this callback is used

Reply:
Can we then use the upstream type so it doesn't look like we're defining this type?
```suggestion
    const saveBlobCallback = (blob: Parameters<Parameters<HTMLCanvasElement['toBlob']>[0]>[0]) => {
```